docs/source/qpu_usage.rst

```diff
@@ -0,0 +1,94 @@
+
+.. note::
+
+    User permissions for QPU access must be enabled by a Forest administrator.  ``QPUConnection`` calls will automatically fail without these user permissions.  Speak to a Forest administrator for information about upgrading your access plan.
+
+Detecting the available QPUs and their structure
+------------------------------------------------
+
+The initialization function for a ``QPUConnection`` object takes a QPU name as its sole argument.  Devices are typically named according to the convention ``[n]Q-[name]``, where ``n`` is the number of active qubits on the device and ``name`` is a human-readable name that designates the device.  The available QPUs can be inspected via a PyQuil interface, as demonstrated in the following snippet:
+
+.. code:: python
+
+    from pyquil.api import get_devices
+    for device in get_devices():
+        if device.is_online():
+            print('Device {} is online'.format(device.name))
+
+The ``Device`` objects returned by ``get_devices`` will capture other characterizing statistics about the associated QPU at a later date.
+
+Execution on the QPU
+--------------------
+
+The user-facing interface to running Quil programs on the QPU is nearly identical to that of the QVM.  A ``QPUConnection`` object provides the following methods:
+
+* ``.run_and_measure(quil_program, qubits, trials=1)``: This method sends the ``Program`` object ``quil_program`` to the QPU for execution, which runs the program ``trials`` many times.  After each run on the QPU, the qubits listed in ``qubits`` are simultaneously measured, and this method returns a list of all of the measurement tuples so obtained.  This call is blocking: it will wait until the QPU returns its results for inspection.
+* ``.run_and_measure_async(quil_program, qubits, trials=1)``: This method has identical behavior to ``.run_and_measure`` except that it is **nonblocking**, and it instead returns a job ID string.
+
+.. note::
+
+    These calls are the only way to send jobs to the QPU at present, and their behavior **does not match** their ``QVMConnection`` counterparts (cf. `Optimized Calls <getting_started.html#optimized-calls>`_).  The ``QVMConnection`` version of ``run`` repeats the execution of a program many times, producing a (potentially) different outcome each time, whereas ``run_and_measure`` executes a program only once and uses the QVM's unique ability to perform wavefunction inspection to multiply sample the same distribution.  The QPU **does not** have this ability, and its ``run_and_measure`` call behaves as the QVM's ``run``.
+
+For example, the following Python snippet demonstrates the execution of a small job on the QPU identified as "8Q-Agave":
+
+.. code:: python
+
+    from pyquil.quil import Program
+    import pyquil.api as api
+    from pyquil.gates import *
+    qpu = api.QPUConnection('8Q-Agave')
```

Comment:
We should probably point to the deployed device in our examples

Reply:
Is there a way to grab a generic deployed device other than sifting through the get_devices() list for one?

Reply:
For now we want to discourage that kind of abstraction to make sure users understand the device they're using fully.

Reply:
Then I didn't / don't understand. You mean this ought to point to Acorn instead?

Reply:
Yes, I believe that's the one we're running with